src/ray/rpc/tests/grpc_server_client_test.cc

```diff
@@ -326,6 +329,178 @@ TEST_F(TestGrpcServerClientFixture, TestTimeoutMacro) {
     std::this_thread::sleep_for(std::chrono::milliseconds(1000));
   }
 }
+class TestGrpcServerClientTokenAuthFixture : public ::testing::Test {
```

Comment:
let's make a standalone test file for this (`grpc_token_auth_test.cc` or something like that)